src/main/java/com/lorepo/icplayer/client/printable/PrintableContentParser.java

```diff
@@ -581,6 +602,15 @@ private static native SplitResult splitModule (String html, int maxHeight, int m
 		return {head: headHTML, tail: tailHTML};
 	}-*/;
 
+	private String getModuleState(String moduleId, String pageId) {
+		if (moduleId == null) return ""; // moduleId == null when printable represents a group
+		String stateKey = pageId + moduleId;

```

Comment:
Sprawdzić czy sklejka tych dwóch argumentów w specyficznym przypadku nie da niejednoznacznego wyniku, np:
moduleId = "ab", pageId = "c" -> stateKey = "abc"
moduleId = "a", pageId = "bc" -> stateKey = "abc"

Reply:
Długość page id jest stała, więc opisywana sytuacja nie powinna mieć miejsca.

Page.java, linia 103
`this.id = UUID.uuid(16);`